spark-streaming-wordcount-java-processor/README.md

```diff
@@ -0,0 +1,62 @@
+Spring XD spark streaming processor example
+=================
+
+### Setup and configure spark cluster
+
+Spark streaming module can be run on any of the supporting cluster modes.
+You can either setup spark `standalone` cluster or use `mesos`, `yarn`. For instructions, see [spark cluster overview] (https://spark.apache.org/docs/1.2.1/cluster-overview.html) and [spark standalone setup] (https://spark.apache.org/docs/1.2.1/spark-standalone.html).
+
+For the development experience, you can use `local` mode as well.
+
+Once the spark cluster is setup, you can configure the property `spark.master` in XD servers.yml to use the corresponding cluster manager URL.The `spark.master` URL property can be configured at the module implementation using `@SparkConfig` annotation that returns Spark configuration properties:
+
+```
+  @SparkConfig
+	public Properties getSparkConfigProperties() {
+		Properties props = new Properties();
+		// Any specific Spark configuration properties would go here.
+		// These properties always get the highest precedence
+		props.setProperty(SPARK_MASTER_URL_PROP, "local[4]");
+		return props;
+	}
+```
+
+By default it is set to use `spark://localhost:7077`
```

Comment:
In both Scala and Java examples - we should explicitly indicate that running Spring XD standalone requires `local` mode to be set for Spark. Will fix during merge